playwright-e2e/tests/book-visit-for-a-remand-prisoner.spec.ts

```diff
@@ -23,7 +23,7 @@ test.describe('Book a visit for remand prisoners', () => {
         expect(name).toBe(prisonerName)
 
     })
-   
+   // This tests also verifies the fuctionality of displaying "visit requested" intead of visit booked. 
     test('Should be able to book a visit for remand prisoner', async ({
```

Comment:
Already merged so I'm a bit late, but if this test is for **requests** then perhaps the description should be updated to `Should be able to **request** a visit...` so that it is clearer what the intent is?